[Java] BAS OnPrem connectivity instructions (#477)
SAP/cloud-sdk: docs/java/features/connectivity/destinations.mdx

#### Destinations With Proxy Type `OnPremise`

> We have to perform some configuration steps to tell the SAP Cloud SDK how to access the SAP BTP destination service and connectivity service.

Reviewer: ```suggestion
To access the **Destination** and **Connectivity** service on SAP BTP we have to configure the SAP Cloud SDK accordingly.
```

:::note Are your client and server located on the same Network?
If your localhost (client) resides in the same network zone as the target system (server) you can access it directly.
Configuration of SAP BTP connectivity services is not needed.
All you have to do is creating a destination via environment variable `destinations`.
:::

##### Developing in Business Application Studio

> If you are developing your application in Business Application Studio (BAS), reaching On-Premise systems can be achieved by performing the following steps:

Reviewer: ```suggestion
If you are developing an application in the **Business Application Studio (BAS)**, and want to reach On-Premise systems, perform the following steps:
```

1. Make sure your BAS instance is part of the same CF space as your Cloud Connector.

> 1. Add two properties called `WebIDEEnabled` and `HTML5.DynamicDestination`, both with value `true`, to your On-Premise destination.

Reviewer: ```suggestion
1. Add properties `WebIDEEnabled` and `HTML5.DynamicDestination` to your On-Premise destination with the value `true`.
```

Reviewer: Maybe having an example here?

Author: You mean like a screenshot?

> 1. In BAS, configure your `destinations` environment variable as follows:

Reviewer: ```suggestion
1. In BAS, configure the environment variable of every On-Premise `destination` as follows:
```

Reviewer: I was confused between plural, possessive case and what exactly relates to what. Check if my suggestion sound like correct for you.

Author: No, that is actually not correct. The SDK looks for one specific environment variable, which is called `destinations`. This variable contains multiple service bindings. The example I provide, however, displays the entire `destinations` variable... So I would actually leave the description as is.

```json
[
  {
    "type": "HTTP",
    "name": "<DESTINATION-NAME>",
    "url": "http://<DESTINATION-NAME>.dest",
    "ProxyHost": "localhost",
    "ProxyPort": 8887
  }
]
```

> Please refer to [the BAS connectivity guide](../../../js/guides/bas-external-system) for more information and a detailed description of the technical background. 

Reviewer: ```suggestion
Please refer to [the BAS connectivity guide](../../../js/guides/bas-external-system) created for the JavaScript SDK to get more information and a detailed description of the technical background. The information there equally applies to the Java SDK.
```
